ui/webui/src/components/Error.jsx

```diff
@@ -80,8 +80,7 @@ export const CriticalError = ({ exception, isBootIso, reportLinkURL }) => {
     const openBZIssue = (reportURL) => {
         setPreparingReport(true);
         cockpit
-                .file("/tmp/webui.log")
-                .replace(logContent)
+                .spawn(["bash", "-c", "journalctl -a > /tmp/webui.log"])
```

Comment:
Aren't we losing the changes from the review with this?

Reply:
Ok, it is nondestructive now.

Reply:
Yes sorry